doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_kubernetes.po

```diff
@@ -217,3 +217,398 @@ msgstr ""
 "<https://docs.conda.io/projects/conda/en/latest/user-"
 "guide/concepts/channels.html>`_。"
 
+#: ../../source/user_guide/deployment_kubernetes.rst:104
+msgid ""
+"Xorbits is now able to integrate `JuiceFS <https://juicefs.com/en/>`_, a distributed POSIX file system that can be easily integrated with Kubernetes to provide persistent storage, as one of the storage backend."
+msgstr ""
+"Xorbits现在集成了 `JuiceFS <https://juicefs.com/zh-cn/>`_  ，一个可以与Kubernetes轻松集成来提供持久化存储的分布式POSIX文件系统。"
```

Comment:
You may want to add a spece between a chinese char and a english char or a number.
```suggestion
"Xorbits 现在集成了 `JuiceFS <https://juicefs.com/zh-cn/>`_  ，一个可以与 Kubernetes 轻松集成来提供持久化存储的分布式 POSIX 文件系统。"
```